dkg/dkg.py

```diff
@@ -323,7 +323,8 @@ def g_i(z):
 #     IA if no message sent
 #
 # Validate:
-# - all expected private keys are revealed
+# - all expected private keys of disqualified members
+#     which have sent valid shares in phase 4 are revealed
```

Comment:
Maybe we can simplify it to:
```
- all private keys of members disqualified after phase 5
   DQ if the number of keys is incorrect
```
? 

Does it reflect what we tried to say here?